decentraland/unity-explorer
# Feat: advanced debug menu

- Implements the requirements of the ticket: https://github.com/decentraland/unity-explorer/issues/1738
- Added feature enable viewing of debug panel on start depending on -debug flag from args or always in editor

### How to Test

1. Launch app without -debug flag, check the debug panel
2. Launch app with -debug flag, check the debug panel again

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
